pkg/token/token.go

```diff
@@ -1,38 +1,52 @@
 package token
 
 import (
+	"encoding/json"
 	"fmt"
 	"io"
 	"net/http"
 	"os"
 	"path/filepath"
 
-	msg "github.com/aziontech/azion-cli/messages/root"
+	"github.com/aziontech/azion-cli/pkg/logger"
+	"github.com/aziontech/azion-cli/utils"
+	"github.com/pelletier/go-toml/v2"
+	"go.uber.org/zap"
 
 	"github.com/aziontech/azion-cli/pkg/config"
 	"github.com/aziontech/azion-cli/pkg/constants"
 )
 
+const settingsFilename = "settings.toml"
+
 type HTTPClient interface {
 	Do(req *http.Request) (*http.Response, error)
 }
 
 type Token struct {
```

Comment:
Maybe we can put these structs in a `models.go` file? (like we're doing for ther APIs)

Reply:
I agree :+1: